<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BudovySearchForm</class>
 <widget class="QWidget" name="BudovySearchForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>LV:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Na parcele:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Č. domovní:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="cisloDomovniLineEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="naParceleLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Zp. využití:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lvBudovyLineEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="mZpVyuzitiCombo"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
